README.md

```diff
@@ -1,6 +1,8 @@
 # Rust IDE built using the IntelliJ Platform
 
-[![Build Status](https://img.shields.io/travis/alexeykudinkin/intellij-rust/master.svg)](https://travis-ci.org/alexeykudinkin/intellij-rust) [![Join the chat at https://gitter.im/alexeykudinkin/intellij-rust](https://img.shields.io/badge/Gitter-Join%20Chat-blue.svg)](https://gitter.im/alexeykudinkin/intellij-rust?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
+[![Build Status Travis](https://img.shields.io/travis/alexeykudinkin/intellij-rust/master.svg)](https://travis-ci.org/alexeykudinkin/intellij-rust)
+[![Build Status Appveyor](https://ci.appveyor.com/api/projects/status/r4fc684iuxda17wj/branch/master?svg=true)](https://ci.appveyor.com/project/matklad/intellij-rust/branch/master)
```

Comment:
you can use [shields.io](http://shields.io/) here too:

[![Build Status Appveyor](https://img.shields.io/appveyor/ci/matklad/intellij-rust/master.svg)](https://ci.appveyor.com/project/matklad/intellij-rust/branch/master)

```md
[![Build Status Appveyor](https://img.shields.io/appveyor/ci/matklad/intellij-rust/master.svg)](https://ci.appveyor.com/project/matklad/intellij-rust/branch/master)
```